torch/testing/_internal/common_methods_invocations.py

```diff
@@ -190,6 +191,7 @@ def __init__(self,
         self.handles_extremals = handles_extremals
         self.handles_complex_extremals = handles_complex_extremals
         self.promotes_integers_to_float = promotes_integers_to_float
+        self.is_sparse_supported = is_sparse_supported
```

Comment:
Also update the OpInfo for an op that supports sparse so it will be tested.

Reply:
This is done